src/base/rss/rssfeed.cpp

```diff
@@ -399,6 +399,22 @@ void Feed::downloadArticleTorrentIfMatching(const ArticlePtr &article)
     BitTorrent::Session::instance()->addTorrent(torrentUrl, params);
 }
 
+QString Feed::generateSavePath(const QString &savePathTpl, const ArticlePtr &article)
+{
+    QRegExp rx("<(.+)>");
```

Comment:
You don't think it's too greedy expression for this? AFAIK, if you have "<cat1> <cat2>" this regexp matches only once with `cap(0) = "<cat1> <cat2>"` and `cap(1) = "cat1> <cat2"`.